runelite-client/src/main/java/net/runelite/client/plugins/dropparty/DropPartyPlugin.java

```diff
@@ -138,8 +138,9 @@ private void onGameTick(GameTick event)
 			{
 				continue;
 			}
-			if (Text.standardize(player.getName()).equalsIgnoreCase(playerName))
+			if (player.getName().equalsIgnoreCase(playerName))
 			{
+				log.error("found running player");

```

Comment:
Doesn't seem like this should be a error message

Reply:
oh oops i left logs in that was mybad, was debugging